src/Sylius/Bundle/AdminBundle/Resources/views/CatalogPromotion/Scope/for_variants.html.twig

```diff
@@ -1,5 +1,5 @@
 {% form_theme field '@SyliusAdmin/Form/theme.html.twig' %}
 
-{{ form_row(field.variants, {'remote_url': path('sylius_admin_ajax_all_product_variants_by_phrase'), 'remote_criteria_type': 'contains', 'remote_criteria_name': 'phrase', 'load_edit_url': path('sylius_admin_ajax_all_product_variants_by_codes')}) }}
+{{ form_row(field.variants, {'remote_url': path('sylius_admin_ajax_all_product_variants_by_phrase', {limit: 25}), 'remote_criteria_type': 'contains', 'remote_criteria_name': 'phrase', 'load_edit_url': path('sylius_admin_ajax_all_product_variants_by_codes')}) }}
```

Comment:
Define a default limit value in the route configuration and remove it from here.
```suggestion
{{ form_row(field.variants, {'remote_url': path('sylius_admin_ajax_all_product_variants_by_phrase'), 'remote_criteria_type': 'contains', 'remote_criteria_name': 'phrase', 'load_edit_url': path('sylius_admin_ajax_all_product_variants_by_codes')}) }}
```

Reply:
I realised you cannot add default value in route config. It's static or dynamic but cannot be default there.
Correct me if I'm wrong.